tests/install_upgrade_operators/daemonset/test_daemonset_params.py

```diff
@@ -19,9 +18,7 @@ def test_no_new_cnv_daemonset_added(is_jira_53226_open, sno_cluster, cnv_daemons
     Since cnv deployments image validations are done via polarion parameterization, this test has been added
     to catch any new cnv deployments that is not part of cnv_deployment_matrix
     """
-    cnv_daemonsets = ALL_CNV_DAEMONSETS.copy()
-    if sno_cluster or not get_hostpath_provisioner():
-        cnv_daemonsets.remove(HOSTPATH_PROVISIONER_CSI)
+    cnv_daemonsets = ALL_CNV_DAEMONSETS.copy() if not sno_cluster else ALL_CNV_DAEMONSETS_NO_HPP_CSI.copy()
```

Comment:
What if we have an SNO cluster with hostpath_provisioner installed? Or a multiple-node cluster without hostpath_provisioner?

Reply:
> What if we have an SNO cluster with hostpath_provisioner installed?